lib/money/money/formatting_rules.rb

```diff
@@ -68,6 +68,11 @@ def translate_formatting_rules(rules)
       rescue I18n::MissingTranslationData
         # Do nothing
       end
+      begin
+        rules[:format] ||= I18n.t :format, scope: "number.currency.format", raise: true
+      rescue I18n::MissingTranslationData
+        # Do nothing
+      end
```

Comment:
Maybe this code could be move to `determine_format_from_formatting_rules` since `:symbol_position`, `:symbol_before_without_space` and `:symbol_after_without_space` are deprecated options.

Something like:
```rb
def determine_format_from_formatting_rules(rules)
  begin
    I18n.t :format, scope: "number.currency.format", raise: true
  rescue I18n::MissingTranslationData
    return currency.format if currency.format && !rules.has_key?(:symbol_position)
    symbol_position = symbol_position_from(rules)
    if symbol_position == :before
      rules.fetch(:symbol_before_without_space, true) ? '%u%n' : '%u %n'
    else
      rules[:symbol_after_without_space] ? '%n%u' : '%n %u'
    end
  end
end
```